src/features/rewards/welcomePage/style.ts

```diff
@@ -71,13 +77,24 @@ export const StyledBackground = styled<{}, 'div'>('div')`
 export const StyledBatLogo = styled<{}, 'div'>('div')`
   margin: 5px auto 0;
   height: 152px;
+
+  @media (max-width: 460px) {
+    height: 100px;
+  }
 `
 
-export const StyledRewardsTitle = styled(Heading)`
+export const StyledRewardsTitle = styled<{}, 'span'>('span')`
```

Comment:
Why was this changed from Heading?

Reply:
I had discussed this with @petemill 

The font size for a couple of the items in the welcome header needs to change based on screen size. Adding this media query override kind of defeats the purpose of using the `Heading` component (which shouldn't be changed). Adding this change to the `Heading` component style could have potentially unrelated consequences, so this seemed the easiest thing to do.

Reply:
you can still use header component like was before and just add media query. This way you use standard component and have custom mobile